app/api/data.py

```diff
@@ -254,47 +254,9 @@ def edit_core_data():
         notify_slack_error(str(e), 'edit_core_data')
         return flask.jsonify(str(e)), 400
 
-    # we construct the batch from the push context
     context = payload['context']
-    flask.current_app.logger.info('Creating new batch from context: %s' % context)
-    batch = Batch(**context)
-    batch.user = get_jwt_identity()
-    batch.isRevision = True
-    db.session.add(batch)
-    db.session.flush()  # this sets the batch ID, which we need for corresponding coreData objects
-
-    # check each core data row that the corresponding date/state already exists in published form
-    core_data_dicts = payload['coreData']
-    core_data_objects = []
-    for core_data_dict in core_data_dicts:
-        flask.current_app.logger.info('Creating new core data row: %s' % core_data_dict)
-
-        # check that there exists at least one published row for this date/state
-        date = core_data_dict['date']
-        state = core_data_dict['state']
-        if not any_existing_rows(state, date):
-            return flask.jsonify("No existing published row for state %s on date %s" % (
-                state, date)), 400
-
-        core_data_dict['batchId'] = batch.batchId
-        core_data = CoreData(**core_data_dict)
-        db.session.add(core_data)
-        core_data_objects.append(core_data)
-
-    db.session.flush()
-
-    json_to_return = {
-        'batch': batch.to_dict(),
-        'coreData': [core_data.to_dict() for core_data in core_data_objects],
-    }
-
-    db.session.commit()
-
-    notify_slack(
-        f"*Pushed batch #{batch.batchId}* (type: {batch.dataEntryType}, user: {batch.shiftLead})\n"
-        f"{batch.batchNote}")
-
-    return flask.jsonify(json_to_return), 201
+    core_data = payload['coreData']
```

Comment:
yesss removing redunancies - thank you! I'm actually thinking we may want to get rid of `/batches/edit` entirely since it's not used anywhere, it's just the `edit_states_daily` method that's used these days (including by edit tools)

Reply:
I'd be fine removing it entirely. Keeping both endpoints around seems like a maintenance/testing headache we'll regret, and I'm not sure what we'd use `/batches/edit` for at this point.

Reply:
I can remove it, it's not a huge deal either way now that they share code